packages/retail-ui-extensions/src/components/Section/Section.ts

```diff
@@ -7,6 +7,7 @@ export interface SectionHeaderAction {
 
 export interface SectionProps {
   title?: string;
+  isLoading?: boolean;
```

Comment:
@heltisace is this actually a thing? I did not know we could display loading indicators like this in Sections/Cards